SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -120,9 +120,11 @@ theorem Nat.one_mod_two_pow_succ_eq {n : Nat} : 1 % 2 ^ n.succ = 1 := by
 
 @[simp]
 lemma ofInt_ofNat (w n : Nat) :
-    BitVec.ofInt w (OfNat.ofNat n) = BitVec.ofNat w n :=
+    BitVec.ofInt w (no_index (OfNat.ofNat n)) = BitVec.ofNat w n :=
   rfl
 
+lemma ofInt_ofNat' : BitVec.ofInt w (OfNat.ofNat (α := ℤ) x ) = x#w := rfl
+
 -- @[simp]
```

Comment:
It seems you didn't remove the primed version, but this one should not be needed after the `no_index` fix, right?

Reply:
Right. Sorry for this and thank you for the post-commit review: https://github.com/opencompl/ssa/pull/new/clean_unneeded_stmt.